E/eccodes/build_tarballs.jl

```diff
@@ -3,18 +3,19 @@
 using BinaryBuilder, Pkg
 
 name = "eccodes"
-version = v"2.18.0"
+version = v"2.19.1"
 
 # Collection of sources required to complete build
 sources = [
-    ArchiveSource("https://confluence.ecmwf.int/download/attachments/45757960/eccodes-2.18.0-Source.tar.gz", "d88943df0f246843a1a062796edbf709ef911de7269648eef864be259e9704e3"),
+    ArchiveSource("https://confluence.ecmwf.int/download/attachments/45757960/eccodes-$version-Source.tar.gz", "9964bed5058e873d514bd4920951122a95963128b12f55aa199d9afbafdd5d4b"),
     DirectorySource("./bundled"),
 ]
 
 # Bash recipe for building across all platforms
 script = raw"""
+version=2.19.1
 cd $WORKSPACE/srcdir
-cd eccodes-2.18.0-Source
+cd eccodes-${version}-Source
 if [ ${target} = "x86_64-w64-mingw32" ] || [ ${target} = "i686-w64-mingw32" ] ; then 
```

Comment:
```suggestion
if [[ ${target} = *-mingw* ]] ; then 
```

Reply:
I had removed the quotes for a reason :slightly_smiling_face:

Reply:
Ah, I didn't see that bit and assumed the issue was with the brackets. I'm still new to this diff syntax and have trouble parsing it.